action.yml

```diff
@@ -107,7 +107,7 @@ runs:
     - uses: actions/setup-node@v3
       if: ${{ inputs.setup-node == 'yes' }}
       with:
-        node-version-file: '.nvmrc'
+        node-version-file: ${{ inputs.working-dir && join([inputs.working_dir, '.nvmrc'], '/') || '.nvmrc' }}
```

Comment:
Using this ternary operator syntax: https://docs.github.com/en/actions/learn-github-actions/expressions#example